index.html

```diff
@@ -18208,11 +18209,26 @@ <h3>
                 "idlAttrType"><code>double</code></span>, readonly
               </dt>
               <dd>
-                The context time of the block of audio being processed. By
-                definition this will be equal to the value of
-                <a>BaseAudioContext</a>'s <a data-link-for=
-                "BaseAudioContext">currentTime</a> attribute that was most
-                recently observable in the <a>control thread</a>.
+                <p>
+                  The context time of the block of audio being processed. By
+                  definition this will be equal to the value of
+                  <a>BaseAudioContext</a>'s <a data-link-for=
+                  "BaseAudioContext">currentTime</a> attribute that was most
+                  recently observable in the <a>control thread</a>.
+                </p>
+                <p>
+                  This must be equal to <a>currentFrame</a> divided by
+                  <a>sampleRate</a>.
+                </p>
+              </dd>
+              <dt>
+                <code><dfn>currentFrame</dfn></code> of type <span class=
+                "idlAttrType"><code>unsigned long long</code></span>, readonly
+              </dt>
+              <dd>
+                The current frame number of the context for the block of audio
+                being processed. This is very closely related to
+                <a>currentTime</a>.
```

Comment:
Yeah, the reason I left currentTime in is because no where in the spec do we talk about currentFrame. By leaving currentTime in, and relating the two, we don't have to define currentFrame except here.  I suppose we could relate currentFrame to context.currentTime elsewhere and remove it from the global scope.

Reply:
@padenot: Anything that needs to be done here?